test/Microsoft.ML.Predictor.Tests/Microsoft.ML.Predictor.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <NoWarn>1701;1702;1705;0618</NoWarn>
```

Comment:
Perhaps if the warning really is inappropriate in one very specific case, we could mark the very specific case rather than saying, "forevermore we will never, under any circumstances, be aware when this warning is generated."